contract/contracts/Voting/Voting.sol

```diff
@@ -104,15 +118,10 @@ contract Voting {
         }
         revert("You are not registered to vote in this proposal");
     }
```

Comment:
I just saw this function, isnt it better if we just pass the array of voters instead of full Proposal struct? i think it will be more efficient since it reduce the memory used, what do u think?

```solidity
function findVoter(Types.Proposal storage proposal, address voterAddress) internal view returns (Types.Member storage) {
    for (uint256 i = 0; i < proposal.voters.length; i++) {
        if (proposal.voters[i].memberAddress == voterAddress) {
            return proposal.voters[i];
        }
    }
    revert("You are not registered to vote in this proposal");
}
```

Reply:
Yes, I'm planning to refactor the whole thing later. For now, in the scope of this PR, I thought I'd keep it to the beacon proxy and solo contract implementation.